cmd/podman/machine/list.go

```diff
@@ -214,8 +224,8 @@ func toHumanFormat(vms []*machine.ListResponse) ([]*machineReporter, error) {
 		response.Created = units.HumanDuration(time.Since(vm.CreatedAt)) + " ago"
 		response.VMType = vm.VMType
 		response.CPUs = vm.CPUs
-		response.Memory = units.HumanSize(float64(vm.Memory) * units.MiB)
-		response.DiskSize = units.HumanSize(float64(vm.DiskSize) * units.GiB)
+		response.Memory = units.HumanSize(float64(vm.Memory))
+		response.DiskSize = units.HumanSize(float64(vm.DiskSize))
```

Comment:
Were these wrong before with multiplication against the units.*?

Reply:
These were not previously wrong, I just moved the multiplication from the rendering code to the data gathering (See machine.go:605) so that the wsl backend could provide smaller values with greater precision. Since WSL is returning usage it's typically in megs instead of gigs for diskspace. It can also now return fractional megs with human rendering.